### PR TITLE
Add Periods to end comments, @params and @returns

### PR DESCRIPTION
**Pull request by ahmadawais**
In accordance with WP PHP documentation standards Add Periods to end comments, @params and @returns

### CODE REVIEW

File: lib/endpoints/class-wp-rest-controller.php [hunk 1]
```diff
@@ -11,121 +11,121 @@ public function register_routes() {
 	}
 
 	/**
-	 * Get a collection of items
+	 * Get a collection of items.
 	 *
 	 * @param WP_REST_Request $request Full data about the request.
-	 * @return WP_Error|WP_REST_Response
+	 * @return WP_Error|WP_REST_Response.
```
Comment (danielbachhuber) [inline lib/endpoints/class-wp-rest-controller.php]: I haven't ever seen periods at the end of the type, only when there's a comment too. Can you clarify where else in core follows this pattern?

Comment (ahmadawais) [inline lib/endpoints/class-wp-rest-controller.php]: Hey, @danielbachhuber !

Periods at the end of the type are the part of standard WordPress PHP documentation in the Core handbook. You can check this information here https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods

Comment (danielbachhuber) [inline lib/endpoints/class-wp-rest-controller.php]: I read:

> @return: Should contain all possible return types, and a description for each. Use a period at the end. Note: @return void should not be used outside of the default bundled themes.

This `@return` doc doesn't have a description, so I don't think it needs a period.

Comment (ahmadawais) [inline lib/endpoints/class-wp-rest-controller.php]: @danielbachhuber  Though it does mention ` Use a period at the end.` 
What do you think about that? Let me know. I'll follow your lead.

Comment (danielbachhuber) [inline lib/endpoints/class-wp-rest-controller.php]: Only add a period if there's actually a description, please. Better yet, add descriptions where they're missing, and then you can add a period to that :)

Comment (ahmadawais) [inline lib/endpoints/class-wp-rest-controller.php]: @danielbachhuber Got it, re-submitting the corrected version.

**Comment by ahmadawais** [comment]
@danielbachhuber Kindly, take a look now.

**Comment by danielbachhuber** [comment]
:ship: :ship: :ship:

**Pull request merged**